dev/src/Helpers.ts

```diff
@@ -363,3 +361,237 @@ export const sortEnrollments = (enrollments: Enrollment[], sortBy: EnrollmentSor
         return enrollmentCompare(a, b, sortBy, descending)
     })
 }
+
+// Class with converter functions for the different proto types
+// TODO(jostein): Move this to a separate file
```

Comment:
Ideally, I would prefer to move it as part of this PR and remove this TODO, if that isn't too much effort.